Dockerfile.windows

```diff
@@ -57,12 +57,18 @@ RUN Write-Host ('Installing Visual C++ Redistributable Package'); `
 
 #
 # Install OpenSSL
+# This must be done after installing Visual Studio
 #
 ADD https://slproweb.com/download/Win64OpenSSL-1_1_1L.msi /local/Win64OpenSSL-1_1_1L.msi
 
-RUN Start-Process msiexec.exe `
-    -ArgumentList '/i C:\local\Win64OpenSSL-1_1_1L.msi ', '/quiet ', `
-    '/norestart ' -NoNewWindow -Wait;
+# Hash copied from https://github.com/slproweb/opensslhashes
+RUN $ExpectedHash = \"1e62e6459835f61c418532a1544e295be5400498b3569aadfb6d0b5d7a1c1dbc\";`
+    $ActualHash = $(Get-FileHash /local/Win64OpenSSL-1_1_1L.msi).Hash.ToLower();`
```

Comment:
The above is a SHA256 hash, right? Want to specify that explicitly with `-Algorithm SHA256`?

Reply:
yeah that's default -- added a comment